include/class.ticket.php

```diff
@@ -2939,7 +2939,7 @@ function postReply($vars, &$errors, $alert=true, $claim=true) {
             $attachments = $cfg->emailAttachments() ? $response->getAttachments() : array();
             //Cc collaborators
             $collabsCc = array();
-            if ($vars['ccs'] && Ticket::checkReply('cc', $vars['emailreply'])) {
+            if ($vars['ccs'] && Ticket::checkReply('cc', $vars['emailreply'], true)) {
```

Comment:
isn't it pointless to have this check? `cc` is hardcoded and its `forced` anyhow.

Reply:
Without that check the collaborator wouldn’t be copied on the postReply template message if the agent opened a ticket on a user’s behalf, added a collaborator while opening, chose ‘do not email alert’, and posted an initial agent response. It throws the replyType off.

Reply:
I understand the purpose of the check but I meant it will always return true (as written) - so why have it anyway?